app/views/clinicfinders/_search_form.html.erb

```diff
@@ -28,8 +28,9 @@
         <div class="col-sm-3 info-form-right hide-label">
           <%= f.select :gestation_days,
                        options_for_select(days_options, patient.last_menstrual_period_now_days),
-                       label: '.',
+                       skip_label: true,
                        autocomplete: 'off' %>
+          <%= f.label :last_menstrual_period_days, t('common.days_along'), class: "sr-only" %>
```

Comment:
this is a great catch and something I never would have ever thought of without your putting in this work. Thank you!